mirror-repos-branch

```diff
@@ -41,6 +41,14 @@ trap '{ export EXT="$?"; rm -rf "${REPOS_DIR}" && exit "${EXT}"; }' EXIT
 
 cd "${REPOS_DIR}"
 
+# Check if we are in the monorepo case
+[ ! -d scripts ] && git clone --recurse-submodules "git@github.com:flatcar/scripts"
+pushd scripts
+if git show "origin/${src_branch}":sdk_container/src > /dev/null 2>/dev/null && [ "$(git show "origin/${src_branch}":.gitmodules 2>/dev/null)" = "" ]; then
```

Comment:
I have no clue what the first `git` invocation is supposed to do. Can I have a comment here about it?

Reply:
I think I'd drop it. LTS-2022 is already using sdk_container. LTS-2021 also did at an end of its lifetime.